Resources/Prototypes/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml

```diff
@@ -65,6 +67,8 @@
     steps: 4
     zeroVisible: true
   - type: Appearance
+  - type: StaticPrice
+    price: 5000
```

Comment:
Artifacts are worth less than this, plus it's mapped into salvage and stations so I think this is way too much.